cobalt/media/base/sbplayer_pipeline.h

```diff
@@ -222,8 +222,10 @@ class MEDIA_EXPORT SbPlayerPipeline : public Pipeline,
   // Whether we should save DecoderBuffers for resume after suspend.
   const bool allow_resume_after_suspend_;
 
-  // Whether we enable batched sample write functionality.
-  const bool allow_batched_sample_write_;
+  // Whether we allow batched sample write functionality.
+  const int audio_batched_sample_write_;
+  // Cobalt allows batched sample write with |audio_batched_sample_write_| > 1.
+  bool allow_batched_sample_write_ = false;
```

Comment:
Do we actually need the boolean? Seems like we could get calculate |max_buffers|  directly by using size of |audio_batched_sample_write_|

Reply:
Fixed it with only using |audio_batched_sample_write_| > 1 to allow multiple samples per write.